packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.Props.ts

```diff
@@ -75,4 +75,12 @@ export interface ITooltipHostProps extends React.HTMLAttributes<HTMLDivElement |
    * Optional class name to apply to tooltip host.
    */
   hostClassName?: string;
+
+  /**
+   * Manual way to display tooltip when the element is not directly focusable but an ancestor is,
+   * so it can pass down that state via this property.
+   * It honors overflow rules and keyboard and mouse interaction, so this just adds another interaction way
+   * delegated on another element.
+   */
+  isFocused?: boolean;
```

Comment:
@cschleiden @micahgodbolt I still think TooltipHost is the place I would expect to find this functionality. I'm trying to wrap up this discussion, so if you guys feel strong about my proposal, let me know and I will try the another approach by making the query-overflow logic available from outside and then I would create a custom host component for my specific case.